.gitignore

```diff
@@ -2,3 +2,6 @@
 *.bin
 *_pycache_*
 port_check.lock
+recipes/natural_language_processing/chatbot/chromedriver*
+models/.locks/
+models/models--BAAI--bge-base-en-v1.5/
```

Comment:
I'm not sure if this is a sustainable approach since you'd have to add every model that gets put into your `models/` dir, right?. I would suggest adding the the entire directory to the .gitignore, but that will likely mean moving the `model_converter` out of that directory.

Reply:
I agre, lets keep models separate from model_converter to separate directory.

Reply:
Rebase has been done, what about this implementation: https://github.com/containers/ai-lab-recipes/pull/132/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R6-R9?
These lines will ignore everything in `models` but explicitly un-ignore `models/Containerfile`, `models/README.md`, and the `models/convert_models/` directory.